distributed/comm/tests/test_ucx_config.py

```diff
@@ -33,10 +33,15 @@ async def test_ucx_config(cleanup):
         "cuda_copy": True,
     }
 
-    with dask.config.set(ucx=ucx):
+    # with dask.config.set(ucx=ucx):
```

Comment:
Whoops, I missed this earlier 

```suggestion
```

Reply:
Another great catch, thanks for finding this!